service/src/main/java/de/adorsys/psd2/sandbox/portal/testdata/TestDataService.java

```diff
@@ -1,209 +1,294 @@
 package de.adorsys.psd2.sandbox.portal.testdata;
 
 import de.adorsys.psd2.sandbox.portal.testdata.domain.Account;
+import de.adorsys.psd2.sandbox.portal.testdata.domain.Amount;
+import de.adorsys.psd2.sandbox.portal.testdata.domain.Balance;
+import de.adorsys.psd2.sandbox.portal.testdata.domain.PsuData;
 import de.adorsys.psd2.sandbox.portal.testdata.domain.Transaction;
 import java.math.BigDecimal;
 import java.time.LocalDate;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Currency;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
 import org.springframework.stereotype.Service;
 
 @Service
 public class TestDataService {
 
-  public static final String PSU_ID = "PSU-1";
-  public static final String PSU_PASSWORD = "12345";
-  public static final String IBAN_GIRO = "DE94500105178833114935";
-  public static final String IBAN_SAVINGS = "DE96500105179669622432";
-  public static final Currency EUR = Currency.getInstance("EUR");
-  public static final String ACCOUNT_OWNER = "Melanie Klein";
-  public static final String ACCOUNT_ID_GIRO = "9b86539d-589b-4082-90c2-d725c019777f";
-  public static final String ACCOUNT_ID_SAVINGS = "d460057b-053a-490a-a36e-c0c8afb735e9";
+  private static final Currency EUR = Currency.getInstance("EUR");
 
-  private HashMap<String, Account> accounts;
-  private HashMap<String, HashMap<String, Transaction>> transactions;
+  private Map<String, PsuData> psuMap;
 
+  /**
+   * Default Constructor.
```

Comment:
Not necessary

Reply:
I used an initialiser block and wrote a TODO to fix checkstyle